src/interface/report/Results/Character/PlayerInfo/PlayerGearHeader.js

```diff
@@ -1,10 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Link from 'react-router-dom/es/Link';
+
+const ArmoryLink = function (player) {
+  const profile = player.characterProfile;
+  let battleNetUrl = `https://worldofwarcraft.com/en-${profile.region}/character/${profile.realm}/${player.name}`;
```

Comment:
Pretty sure the link format for this changed; the locale should not be the region but static en-us, and the region needs to be part of the char link.